packages/components/src/internal/components/search/EntityFieldFilterModal.tsx

```diff
@@ -179,9 +190,13 @@ export const EntityFieldFilterModal: FC<Props> = memo(props => {
             queryLabels[parentValue] = label;
         });
         const filterErrors = getFieldFiltersValidationResult(validDataTypeFilters, queryLabels);
-        if (!filterErrors) onFind(entityDataType.instanceSchemaName, validDataTypeFilters);
-        else setFilterError(filterErrors);
-    }, [onFind, validDataTypeFilters]);
+        if (!filterErrors) {
```

Comment:
If we only care about successful Find clicks, wouldn't it be cleaner to put the metric update in the parent onFind?

Reply:
Do we want to track how many times a schemaquery is used to find child samples?

Reply:
Good call on the first part, move this `incrementClientSideMetricCount` for `filterModalAppy` up to the `SampleFinderSection`.

For the second part, I don't think we have a good way to use the client side metrics to track any per SchemaQuery info.